src/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Rekupe Digital Agency</title>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1" name="viewport">
+
+    <link href="styles/reset.css" rel="stylesheet">
+    <link href="styles/main.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles/media.css">
+    <link href="/src/assets/novi-icon.svg" rel="icon" type="image/x-icon">
+    <link href="https://fonts.googleapis.com" rel="preconnect">
+    <link crossorigin href="https://fonts.gstatic.com" rel="preconnect">
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com" rel="preconnect">
+    <link crossorigin href="https://fonts.gstatic.com" rel="preconnect">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+
+<div class="ref-img">
+
+</div>
+
+<header>
+    <div class="container-outer header-container">
+        <nav>
+            <div class="container-inner navbar-container">
+                <ul class="nav-items">
+                    <li><a href="#">Case Studies</a></li>
+                    <li><a href="#">Services</a></li>
+                    <li><a href="#">Contact Us</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="container-inner header-copy">
+            <h1>Your website is an investment in the succes of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <a class="primary-bttn" href="">Let's work together</a>
+        </div>
+
+        <div class="container-inner header-icon">
+            <div class="icon-wave-stroke"></div>
+        </div>
+    </div>
+</header>
+
+<div class="rotation-element"></div>
+
+<!--MAIN-->
+<main>
+    <section class="container-outer">
+        <span class="background-text purple"><h4>hello</h4></span>
+        <div class="container-inner introduction-container">
+            <h3>Rekupe is a digital agency in Los Angeles. <br><br> We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.
+                <br><br> We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+            </h3>
+            <p>Have a question? <a href="#">Click here</a> to contact us!</p>
+        </div>
+    </section>
+
+    <!--CASES-->
+    <section class="container-outer cases-container">
+        <span class="background-text white"><h4>WORK</h4></span>
+
+        <!--Case 1-->
+        <div class="container-inner case-container">
+
+            <div class="align-right" >
+                <img alt="" src="/src/assets/work-image-01.jpeg">
```

Comment:
De relatieve paden kloppen niet
```suggestion
                <img alt="" src="./assets/work-image-01.jpeg">
```